test/WebJobs.Script.Tests/Description/Node/NodeFunctionInvokerTests.cs

```diff
@@ -62,14 +68,27 @@ public void NormalizeBindingData_TypeHandling()
                 { "TestProp4", inputObject2 },
                 { "TestProp5", objectArray },
                 { "TestProp6", null },
-                { "TestProp7", new HttpConfiguration() }
+                { "TestProp7", new BlobProperties { ContentType = "application/json", ContentMD5 = "xyz" } },
```

Comment:
Objects like BlobProperties and PartitionContext are now turned into objects (to a depth of 1), allowing their properties to be accessed.